spec/models/title_spec.rb

```diff
@@ -1,5 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe Title, :type => :model do
```

Comment:
Plus team doesn't like hash rockets for whatever reason. Before you commit be sure to do a quick search for any `=>` in your code and replace them with the other syntax e.g. `type: :model`.

Reply:
Thanks, missed that one.